data/pages-content/sponsor/information.md

```diff
@@ -56,126 +53,130 @@ For details, please refer to the relevant sections below.
 - **Assign the tickets** to every team member that you have chosen to attend the
   conference. You can assign them by changing your order details after you make
   the order. Click the order URL in your order confirmation email from
-  [support@pretix.eu](mailto:support@pretix.eu) with the **Subject: Your
+  [support@pretix.eu](mailto:support@pretix.eu) with the Subject: **Your
   order**: **_XXXX_** and amend accordingly.
 
-**_Deadline_** for registering and assigning the tickets: **_June 27_**
+**Deadline** for registering and assigning the tickets: **2023-06-19**
```

Comment:
There is also another date format below, I think, maybe it would be nice to stick with one format everywhere.
I personally like `19 June, 2023` more, since I've always confuse people from the States when write `2023-06-06`, for example

Reply:
That's a good point! I have now changed all the date format in the individual sections to the `19 June, 2023`! (I hope...)